gammapy/irf/rad_max.py

```diff
@@ -24,3 +31,51 @@ class RadMax2D(IRF):
 
     tag = "rad_max_2d"
     required_axes = ["energy", "offset"]
+
+
+    @classmethod
+    def from_irf(cls, irf: Union[EffectiveAreaTable2D, EnergyDispersion2D]):
+        '''
+        Create a RadMax2D instance from another IRF component.
+
+        This reads the RAD_MAX metadata keyword from the irf and creates
+        a RadMax2D with a single bin in energy and offset using the
+        ranges from the input irf.
+
+        Parameters
+        ----------
+        irf: EffectiveAreaTable2D or EnergyDispersion2D
```

Comment:
Please add backticks to the class names here, such that Sphinx links to them in the docs..